corehq/apps/indicators/models.py

```diff
@@ -48,8 +48,8 @@ def __init__(self, _d=None, **kwargs):
         self.class_path = self._class_path
 
     def __str__(self):
-        return "\n\n%(class_name)s - Modified %(last_modified)s\n %(slug)s, domain: %(domain)s," \
-            " version: %(version)s, namespace: %(namespace)s. ID: %(indicator_id)s." % {
+        return u"\n\n%(class_name)s - Modified %(last_modified)s\n %(slug)s, domain: %(domain)s," \
```

Comment:
Ah. you're right. I missed the parens https://github.com/dimagi/commcare-hq/pull/15328/files#diff-21bd1c0e52720275d9cf19e3dd556434R1039